app/controllers/admin/users_controller.rb

```diff
@@ -1,20 +1,16 @@
 module Admin
   class UsersController < Admin::BaseController
     before_action :set_user, only: %i[show edit destroy]
+    helper_method :sort_column, :sort_direction
 
     def index
-      @filterrific = initialize_filterrific(
-        User.client,
-        params[:filterrific],
-        select_options: { sorted_by: User.options_for_sorted_by }
-      ) || return
-      @users = @filterrific.find.page(params[:page])
-      respond_to do |format|
-        format.html
-        format.js
-      end
-    rescue ActiveRecord::RecordNotFound
-      redirect_to(reset_filterrific_url(format: :html)) && return
+      return @users = User.search(params[:search]) if params[:search]
```

Comment:
Can we refactor this like?

@users = User
@users = @users.search(params[:search]) if params[:search].present?
@users = @users.order(sort_column => sort_direction) if sort_column.present? && sort_direction.present?

Make sure to test thoroughly.

Reply:
we can but for search only it was must be returned I tested thoroughly.